sigmastate/src/main/scala/org/ergoplatform/ErgoScriptPredef.scala

```diff
@@ -18,8 +18,7 @@ object ErgoScriptPredef {
   import sigmastate.interpreter.Interpreter._
 
   def compileWithCosting(env: ScriptEnv, code: String, networkPrefix: NetworkPrefix)(implicit IR: IRContext): Value[SType] = {
-    val compiler = new SigmaCompiler(CompilerSettings(
-      networkPrefix, TransformingSigmaBuilder, lowerMethodCalls = true))
+    val compiler = new SigmaCompiler(networkPrefix)
```

Comment:
Unused imports not fixed after this transformation

Reply:
fixed